src/libs/actions/Policy.js

```diff
@@ -438,6 +441,31 @@ function setWorkspaceErrors(policyID, errors) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {errors});
 }
 
+/**
+ * @param {String} policyID
+ */
+function removeUnitError(policyID) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {
+        customUnits: {
+            distance: {
+                errors: null,
+            },
+        },
+    });
+}
+
+/**
+ * Checks if we have any errors stored within the policy custom units.
+ * @param {Object} policy
+ * @returns {Boolean}
+ */
+function hasCustomUnitsError(policy) {
+    if (!policy || !policy.customUnits || !policy.customUnits.distance || !policy.customUnits.distance.errors) {
```

Comment:
Use loadash to simplify this statement. 
Untested but i think you could just have it like so,
```
const errors = lodashGet(policy, ['customUnits', 'distance', 'errors'], '');
return !_.isEmpty(errors);
```
you can use `lodashGet(policy, 'customUnits.distance.errors', '')` too